build.gradle

```diff
@@ -12,14 +12,14 @@ plugins {
 	id 'checkstyle'
 	id 'jacoco'
 	id 'pmd'
-	id 'com.github.ben-manes.versions' version '0.27.0'
+	id 'com.github.ben-manes.versions' version '0.25.0'
 	id "info.solidsoft.pitest" version '1.4.5'
-	id 'io.spring.dependency-management' version '1.0.8.RELEASE'
-	id 'org.owasp.dependencycheck' version '5.2.4'
+	id 'io.spring.dependency-management' version '1.0.9.RELEASE'
+	id 'org.owasp.dependencycheck' version '5.3.0'
 	id 'org.sonarqube' version '2.8'
-	id 'org.springframework.boot' version '2.1.4.RELEASE'
-	id "org.flywaydb.flyway" version "6.1.1"
-	id 'au.com.dius.pact' version '4.0.3'
+	id 'org.springframework.boot' version '2.2.4.RELEASE'
```

Comment:
any reason why the dependency versions downgraded ?

Reply:
It seems Profile Sync's Demo branch did not have latest versions

Reply:
but why you are downgrading the versions of dependencies?

Reply:
This has been addressed in the following commits: https://github.com/hmcts/rd-profile-sync/pull/181/commits/5338b31bf1246beb5f9d4302a78f4830dc0e4372 & https://github.com/hmcts/rd-profile-sync/pull/181/commits/8cf58e15e02afb1a2f6452cb589eacc6bff12bb3